src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -169,6 +169,26 @@ internal static IEnumerable<string> GetCommaSeparatedValues(string text)
             bool targetingAttributes,
             ErrorSink errorSink)
         {
+            if (!targetingAttributes &&
+                string.Equals(
+                    name,
+                    TagHelperDescriptorProvider.CatchAllDescriptorTarget,
+                    StringComparison.OrdinalIgnoreCase))
+            {
+                // '*' as the entire name is ok if we're not targeting attributes because it's the TargetElement
+                // catch-all case.
```

Comment:
Slightly confusing. Suggest `// '*' as the entire name is OK in the TargetElement catch-all case.`